src/Security/Authorization/Voter/UserAclVoter.php

```diff
@@ -25,20 +25,30 @@ public function supportsClass($class): bool
         return is_subclass_of($class, UserInterface::class);
     }
 
+    /**
+     * @param mixed $attribute
+     *
+     * @return bool
```

Comment:
You can use native type

Reply:
Same comment as above, next Pr is about native types